frontend/src/environments/environment.ts

```diff
@@ -4,8 +4,8 @@
 
 export const environment = {
     production: false,
-    webApiUrl: 'https://localhost:7133',
-    webUrl: 'https://localhost:7133',
+    webApiUrl: 'http://localhost:5197',
```

Comment:
Why do you change this?

Reply:
Beacause it wasn't working on that port on my pc

Reply:
Then you need to change swager config to start on a selected port. Right now if we merge this the app will not work on anyone else's computer

Reply:
Changed it back then